window.py

```diff
@@ -1,18 +1,20 @@
-#python3.12 window.py
-#pyinstaller --onefile window.py
-#pyinstaller window.spec
+# TODO: turn into run.bat and build.bat (or .sh for linux?)
+# python3.12 window.py
+# pyinstaller --onefile window.py
+# pyinstaller window.spec
 import PySimpleGUI as gui
 import logging
-from samase import Samase, OpType
+import samase
 import os
 
 LOGFILE = "sezam.log"
 
-localSamase = None
-unpackSamase = None
+# TODO: supply intelligent defaults
+local_samase = ""
+unpack_samase = ""
```

Comment:
Path is the only argument that really changes due to UI, which is why we store it as state, which was previously represented as class. This is state that persists between iterations of the event loop, which is unlike is64, which just reads from `values`, which is available on every iteration of the loop (which is why we can just straight up pass it as a function arg without storing it in a temporary across loop boundary variable).